app/src/main/res/layout/view_user_profile.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="@dimen/profile_padding_vertical_default"
+    android:paddingStart="@dimen/profile_padding_horizontal_default"
+    android:paddingEnd="@dimen/profile_padding_horizontal_default"
+    android:paddingBottom="@dimen/profile_padding_vertical_default"
+    android:dividerPadding="@dimen/profile_padding_divider" >
+
+    <Spinner
+        android:id="@+id/sex_spinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/profile_sex_prompt"/>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingStart="@dimen/profile_padding_spinner_offset"
+        android:paddingEnd="@dimen/profile_padding_spinner_offset" >
+        <EditText
+            android:id="@+id/birthday_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:editable="false"
+            android:hint="@string/profile_birthday_prompt"/>
```

Comment:
Still needs to be fixed.

Reply:
Yeah, I know. It'll get done today or tomorrow. You said to make this a TextView and skin it like an EditText?

Reply:
Yep! You should be able to use [this gorgeous example](https://github.com/marverenic/Jockey/blob/af3d9230a4afa34067a572352edd8620e4d54e58/app/src/main/res/layout/instance_rule.xml#L59-L81) a reference.